qcodes/instrument_drivers/Lakeshore/Model_325.py

```diff
@@ -1,4 +1,4 @@
-import numpy as np
+from enum import IntFlag
```

Comment:
@liangosc so what do you think about use of `IntFlag` vs `Flag` classes? I read in the python docs (or somewhere else?) that although `IntFlag` exists, preference should be given to use of `Flag` and the reason for this is that `Flag` is less forgiving to integer values it can represent - for example, if `Status` was a subclass of `Flag`, it'd will raise an exception on `total_status = Status(sum_of_codes)` if `sum_of_codes` contains codes that are not in the enumeration, for example, if `sum_of_codes == 16+2`. imho, it's a valuable feature in this case. What's your opinion?

Reply:
I agree it''ll be nice to raise an exception when the sum contains codes that are not in the enumeration -- one situation is, the manufacturer may add more error codes in the future and `Flag` will be able to capture it.

However, I like the fact that `IntFlag` supports `==, <, >`. It doesn't seem we have used this feature in the code, but we may have: in the test function, we won't be able to do `st.sampled_from(list(STATUS))` if STATUS is `Flag` type. (It may still work after some modification though.)